src/CoverageTransformer.js

```diff
@@ -97,7 +98,9 @@ export default class CoverageTransformer {
 
 		if (!rawSourceMap) {
 			/* We couldn't find a source map, so will copy coverage after warning. */
-			this.warn(new Error(`Could not find source map for: "${filePath}"`));
+			if (this.warnMissingSourceMap) {
```

Comment:
I see why this failed the test, it should be this.warnMissingSourceMaps, right?

Reply:
Thanks, I fixed the typo.